src/Avalonia.Controls/Notifications/SimpleNotification.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Avalonia.Controls.Notifications
+{
+    /// <summary>
+    /// Defines content for a <see cref="Notification"/> control.
+    /// </summary>
+    /// <remarks>
+    /// This notification content type is compatible with native notifications.
+    /// </remarks>
+    public class SimpleNotification : INotification
```

Comment:
I'm not a huge fan of terms like "Simple" in an API - "simple" in what sense? It doesn't feel very descriptive.

Does calling it `NativeNotification` not make sense?

Reply:
Called it notification and the control NotificationCard